JSON::XS::to_json has been renamed to encode_json

diff -dupr davemp-1.23.orig/lib/Davemp.pm davemp-1.23/lib/Davemp.pm
--- davemp-1.23.orig/lib/Davemp.pm	2010-01-29 18:37:51.652969294 +0100
+++ davemp-1.23/lib/Davemp.pm	2010-01-29 18:38:38.669968654 +0100
@@ -68,7 +68,7 @@ sub handle_request {
             my $vars = $cgi->Vars;
             map { $vars->{$_} =~ s/\%([A-Fa-f0-9]{2})/pack('C', hex($1))/seg; } keys %$vars;            
             my $result = $self->sendCommand($command, $vars);
-            my $jsonstring = to_json($result);
+            my $jsonstring = encode_json($result);
             
             print "HTTP/1.1 200 OK\n";
             #print "Content-type: application/json\n";
